### PR TITLE
feat: add settings

### PR DESCRIPTION
**Pull request by henriettemoe**
make it possible to change modes

### CODE REVIEW

File: src/h5p-vocabulary-drill.ts [hunk 1]
```diff
@@ -7,21 +7,164 @@ import { H5P, H5PContentType, registerContentType } from "h5p-utils";
...
+
+    const buttonAnswerMode = document.createElement("button");
+    buttonAnswerMode.addEventListener("click", handleAnswerModeChange);
+    buttonAnswerMode.addEventListener("click", () =>
+      settings.classList.toggle("visible"),
+    );
+    buttonAnswerMode.appendChild(buttonAnswerModeLabel);
+
+    // TODO: translate
+    const nodeLanguageMode = document.createTextNode("Change language");
```
Comment (boyum) [inline src/h5p-vocabulary-drill.ts:102]:
```suggestion
    // TODO: Show only if `allow changing language` is turned on in semantics
    const nodeLanguageMode = document.createTextNode("Change language");
```

File: src/h5p-vocabulary-drill.ts [hunk 2]
```diff
@@ -7,21 +7,164 @@ import { H5P, H5PContentType, registerContentType } from "h5p-utils";
...
+    container.appendChild(buttonAnswerMode);
+    container.appendChild(buttonLanguageMode);
+
+    settings.appendChild(top);
+    settings.appendChild(container);
+
+    return settings;
+  }
+
+  private async handleAnswerModeChange(): Promise<void> {
```
Comment (boyum) [inline src/h5p-vocabulary-drill.ts:122]: Why async?

File: src/h5p-vocabulary-drill.ts [hunk 3]
```diff
@@ -7,21 +7,164 @@ import { H5P, H5PContentType, registerContentType } from "h5p-utils";
...
+    VocabularyDrill.removeRunnable(this.wrapper);
+    VocabularyDrill.addRunnable(
+      this.wrapper,
+      this.contentId,
+      this.params,
+      newAnswerMode,
+    );
+  }
+
+  private async handleLanguageModeChange(): Promise<void> {
```
Comment (boyum) [inline src/h5p-vocabulary-drill.ts:137]: Why async?

File: src/h5p-vocabulary-drill.ts (hunk 2 above)
Comment (henriettemoe) [reply src/h5p-vocabulary-drill.ts:123]: Good point, don't need it so I'll change it

**Review by boyum — APPROVED** [review]
✨

**Pull request merged**